lib/logstash/outputs/dynatrace.rb

```diff
@@ -74,6 +74,9 @@ class PluginInternalQueueLeftoverError < StandardError; end
       # Include body in debug logs when HTTP errors occur. Body may be large and include sensitive data.
       config :debug_include_body, validate: :boolean, default: false
 
+      # Maximum size payload to send to the Dynatrace API. Batches of events which would be larger than max_payload_size when serialized will be split into smaller batches of events.
```

Comment:
Is this something we want to make configurable or should we make it constant? Making it configurable means users could change it if the Dynatrace API limits change without having to wait for us to release